sunpy/io/jp2.py

```diff
@@ -71,6 +72,101 @@ def get_header(filepath):
 
     return [FileHeader(pydict)]
 
+def header_to_xml(header):
+    """
+    Converts image header metadata into an XML Tree that can be inserted into
+    a JP2 file header.
+
+    Parameters
+    ----------
+    header : `MetaDict`
+        A header dictionary.
+    """
+    # glymur uses lxml and will crash if trying to use
+    # python's builtin xml.etree
+    import lxml.etree as ET
```

Comment:
Will `lxml` be installed on a users PC? I forget if we pull it in from a dependency.

Reply:
iirc Glymur has it as a dep, but if we are importing it directly perhaps we should add it to our deps as well?

Reply:
Yes because it's a dependency of glymur.
Should I also add it as a dependency for sunpy?

Reply:
We should, do we have an io group to add it to in setup.cfg?